packages/dos-donts-block/src/settings.ts

```diff
@@ -269,6 +269,7 @@ export const settings = defineSettings({
             id: 'imagesSection',
```

Comment:
I think this empty section shouldn't be displayed on the initial release

![Screenshot 2023-07-25 at 13 05 23](https://github.com/Frontify/guideline-blocks/assets/9362990/c711c627-5787-4962-bf63-14f3b09d3d11)

Reply:
Good catch, fixed